seedlet-blocks/block-template-parts/header.html

```diff
@@ -1,9 +1,13 @@
-<!-- wp:image {"align":"center","width":128,"height":128,"sizeSlug":"large","className":"is-style-rounded"} -->
-<div class="wp-block-image is-style-rounded"><figure class="aligncenter size-large is-resized"><img src="https://cldup.com/B9dfntUFJE.png" alt="" width="128" height="128"/></figure></div>
```

Comment:
I also removed the rounded style from this image, since we removed that in the main theme.